server/src/api/v1/cards.js

```diff
@@ -0,0 +1,5 @@
+const cardsController = require('../../controllers/v1').cards
+
+module.exports = (api) => {
+  api.post('/cards', cardsController.create)
+}
```

Comment:
@datarockets/cuties the initial thought was that `src/api` is a folder for controllers. Sorry for the misunderstanding, @nikitasakov can we rename it to `src/controllers` in this PR? cause it something that should be here

Reply:
@AleksSenkou I've already created `src/controllers` folder, and as I understand `src/api` is a folder with routes. I think we can rename `src/api` to `src/routes`. What do you think?

Reply:
@nikitasakov good to me 😉